src/js/components/stock-movement-wizard/StockMovement.jsx

```diff
@@ -31,6 +31,7 @@ class StockMovements extends Component {
   }
 
   componentDidMount() {
+    this.props.fetchCurrentLocation();
```

Comment:
is this necessary, I think it's already fetched in the location chooser

Reply:
It doesn't work without it so it is necessary